### PR TITLE
Remove partition id from gRPC responses

### LINKED ISSUE #1455: Remove partition id from client responses
With the implementation of #1037 the user does not need to know which partition an entity was created on. Remove all partition ids from gRPC responses.

### PR DESCRIPTION
**Pull request by menski**
closes #1455

### CODE REVIEW

File: clients/java/src/main/java/io/zeebe/client/api/events/WorkflowInstanceEvent.java [hunk 1]
```diff
@@ -26,9 +26,6 @@
   /** Version of the workflow which this instance was created for */
   int getVersion();
 
-  /** Partition on which the workflow instance was created; */
-  int getPartitionId();
```
Comment (Zelldon) [inline clients/java/src/main/java/io/zeebe/client/api/events/WorkflowInstanceEvent.java]: Maybe the user wants to know where the WF instance ended? What do you think ? But then we need to move the constants into the protocol module for example, to extract the partition id from the returned key

**Review by ChrisKujawa — COMMENTED** [review]
Looks good. I have one question see below

File: clients/java/src/main/java/io/zeebe/client/api/events/WorkflowInstanceEvent.java (hunk 1 above)
Comment (menski) [reply clients/java/src/main/java/io/zeebe/client/api/events/WorkflowInstanceEvent.java]: > Maybe the user wants to know where the WF instance ended?

I'm not sure if the user should care on the client response, in the end it is possible to get that information from exported data, i.e. in Operate.

> But then we need to move the constants into the protocol module for example

Which protocol module do you mean? It is already in the broker protocol, but that module is not accessible from the client.
https://github.com/zeebe-io/zeebe/blob/94bc4d6a165896b46d5aefc10789c34c36242ade/protocol/src/main/java/io/zeebe/protocol/Protocol.java#L37

Comment (Zelldon) [reply clients/java/src/main/java/io/zeebe/client/api/events/WorkflowInstanceEvent.java]: > I'm not sure if the user should care on the client response, in the end it is possible to get that information from exported data, i.e. in Operate.

Maybe you're right. We can leave it as it is.


> Which protocol module do you mean? It is already in the broker protocol, but that module is not accessible from the client.

This is new to me, that this is in the protocol.

**Review by ChrisKujawa — APPROVED** [review]

**Pull request merged**